guides/common/attributes-base.adoc

```diff
@@ -77,8 +77,8 @@
 :oVirtEngine: {oVirt} Engine
 :oVirtShort: {oVirt}
 :package-clean: dnf clean
-:package-install-project: dnf install
-:package-install: dnf install
+:project-package-install: dnf install
+:client-package-install: dnf install
```

Comment:
```suggestion
:client-package-install-el: dnf install
:client-package-install-deb: apt-get install
```
Perhaps this would be more helpful?

Reply:
@maximiliankolb @ekohl Opinions?

Reply:
@maximiliankolb Yes, please, bring it to the meeting. I don't understand very well what you have in mind.